addons/stock/wizard/stock_replenishment_info.py

```diff
@@ -20,6 +20,17 @@ class StockReplenishmentInfo(models.TransientModel):
     json_lead_days = fields.Char(compute='_compute_json_lead_days')
     json_replenishment_history = fields.Char(compute='_compute_json_replenishment_history')
 
+    warehouseinfo_ids = fields.One2many(related='orderpoint_id.warehouse_id.resupply_route_ids')
+    wh_replenishment_option_ids = fields.One2many('stock.replenishment.option', 'replenishment_info_id', compute='_compute_wh_replenishment_options')
+
+    @api.depends('orderpoint_id')
+    def _compute_wh_replenishment_options(self):
+        for replenishment_info in self:
+            replenishment_info.wh_replenishment_option_ids = self.env['stock.replenishment.option'].create([
+                {'product_id': replenishment_info.product_id.id, 'route_id': route_id.id, 'replenishment_info_id': replenishment_info.id}
```

Comment:
In fact you anyway needs to create them since you have a button on the line and our framework need real record to enable button action. So it's perfect that way